samples/nrf9160/gnss/Kconfig

```diff
@@ -6,12 +6,43 @@
 
 menu "GNSS sample"
 
+choice
+	default GNSS_SAMPLE_MODE_CONTINUOUS
+	prompt "Select GNSS operation mode"
+
+config GNSS_SAMPLE_MODE_CONTINUOUS
+	bool "Continuous tracking"
+
+config GNSS_SAMPLE_MODE_PERIODIC
+	bool "Periodic fixes"
+
+endchoice
+
+if GNSS_SAMPLE_MODE_PERIODIC
+
+config GNSS_SAMPLE_PERIODIC_INTERVAL
+	int "Fix interval for periodic fixes"
+	range 10 65535
```

Comment:
There was some limitation wrt. interval on older MFWs, right? Should we mention it here or in some other place?

Reply:
Yes, there is. As discussed offline, that limitation is documented in the GNSS API and I don't think we need to repeat the same thing here.